client/src/cl_parse.cpp

```diff
@@ -801,6 +801,13 @@ static void CL_LoadMap(const odaproto::svc::LoadMap* msg)
 
 	if (!missingfiles.empty())
 	{
+		if (::missingCommercialIWAD)
+		{
+			Printf(PRINT_WARNING, "Server requires commercial IWAD that was don't have.\n");
```

Comment:
"Server requires commercial IWAD that was don't have."

Should this be: "Server requires commercial IWAD that **we** don't have." ?

Reply:
I would think it should be "Server requires commercial IWAD that **you** don't have." or "Server requires commercial IWAD that **wasn't found**."

Reply:
Fixed.